Feedback/forms.py

```diff
@@ -8,17 +12,15 @@
     ('5', ''),
 )
 
-		
-class FeedbackForm(forms.Form):
-	Punctuality = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Subject_knowledge = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Behavior = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Method_of_teaching = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Audibility = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
-	Syllabus_coverage = forms.IntegerField(label='', widget=forms.RadioSelect(
-		choices=choices), required=True)
+class FeedbackForm(forms.ModelForm):
+	class Meta:
+		model = Feedback
+		exclude = ['teacher', 'section']
+		widgets = {
+		'Punctuality' : forms.RadioSelect(choices=choices, ),
```

Comment:
Give four spaces before `Punctuality` and in following attributes upto `Syllabus_coverage`